dev/MRTCore/mrt/MrtCore.sln

```diff
@@ -168,6 +170,20 @@ Global
 		{3C618444-4B80-492E-8972-BFAEF8700A52}.Release|x64.Build.0 = Release|x64
 		{3C618444-4B80-492E-8972-BFAEF8700A52}.Release|x86.ActiveCfg = Release|Win32
 		{3C618444-4B80-492E-8972-BFAEF8700A52}.Release|x86.Build.0 = Release|Win32
+		{4243B8DA-BDB5-4D65-B1D5-EEED375FAD13}.Debug|ARM.ActiveCfg = Debug|Win32
```

Comment:
Why do we need this change?

Reply:
The Insights lib project needs to be a part of the solution for it to be referenced by an existing project in the solution. I mean, I found that I could only add as a reference a project that was already a part of the solution.

Reply:
This further speaks in favor of having it as header-only

Reply:
Having it header-only will further increase the disk footprint.

Reply:
> Having it header-only will further increase the disk footprint.

By how much? As I mentioned above, I would expect the linker to take care of most/all of this. To make a call here we need to see the actual numbers.

Reply:
https://github.com/microsoft/WindowsAppSDK/pull/1268 switched impl to header-only